src/cf-forms/src/atoms/select.less

```diff
@@ -5,7 +5,9 @@
 
     select {
         width: 100%;
-        padding: unit( 8px / @base-font-size-px, em )
+        line-height: @base-line-height;
+        padding: unit( 7px / @base-font-size-px, em )
+                 unit( 6px / @base-font-size-px, em )
```

Comment:
Isn't the 6px duplicative here?

Reply:
No, the first one is left/right, the second one is bottom. I had hoped 7 would work for both top and bottom, but it renders out at 36px instead of 35px.

Reply:
@wpears correct, it's top (7px), left/right (6px), bottom (6px). If there wasn't a specific bottom setting (the second 6px), it'd only be top/bottom (7px), left/right (6px) and would result in 36px in height.

Reply:
Oh, of course, the base line height is now 1.375, so 16*1.375 = 22 + 13 = 35

Sorry for the detritus